### PR TITLE
break apart rendering into separate functions

### PR DESCRIPTION
**Pull request by zandrmartin**
please review, this works but I'm not sure it's the perfect implementation. in particular I don't know if every `request_rerender()` is asking for the correct panels.

### CODE REVIEW

File: src/ui.c [hunk 1]
```diff
@@ -134,43 +134,102 @@ void fetch_pending() {
...
+		.y = 1,
+		.width = tb_width() - config->ui.sidebar_width,
+		.height = tb_height() - 2
+	};
+	state->panels.message_list = geo;
+	render_items(geo);
+}
+
+static void rerender_message_view() {
+}
```
Comment (zandrmartin) [inline src/ui.c]: This was just here for the future, when we actually implement the message view. But I can remove it for now if you'd prefer.

File: include/state.h [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #include "bind.h"
 #include "util/list.h"
 #include "worker.h"
+// #include "render.h"
```
Comment (zandrmartin) [inline include/state.h]: Doh. Will remove.

File: src/ui.c (hunk 1 above)
Comment (SirCmpwn) [inline src/ui.c]: Please do.

File: include/state.h [hunk 3]
```diff
@@ -46,7 +47,7 @@ struct aerc_state {
 	size_t selected_account;
 	list_t *accounts;
 	bool exit;
-	bool rerender;
+	unsigned int rerender;
```
Comment (SirCmpwn) [inline include/state.h]: Bleh, alright.

**Pull request merged**